plugin/pxr/pxr/pxr.h.in

```diff
@@ -1,60 +0,0 @@
-//
```

Comment:
Thanks for removing this! I wasn't sure why it was still here either.

Reply:
I went through and added pxr.h everywhere the namespace stuff was in use.  And yeah, I think the pxr.h.in in THIS repo was probably just leftover cruft...